include/nbl/builtin/hlsl/scan/virtual_workgroup.hlsl

```diff
@@ -0,0 +1,93 @@
+#ifndef _NBL_HLSL_SCAN_VIRTUAL_WORKGROUP_INCLUDED_
+#define _NBL_HLSL_SCAN_VIRTUAL_WORKGROUP_INCLUDED_
+
+// TODO (PentaKon): Decide if these are needed once we have a clearer picture of the refactor
+#include "nbl/builtin/hlsl/limits/numeric.hlsl"
+#include "nbl/builtin/hlsl/math/typeless_arithmetic.hlsl"
+#include "nbl/builtin/hlsl/workgroup/arithmetic.hlsl" // This is where all the nbl_glsl_workgroupOPs are defined
+#include "nbl/builtin/hlsl/scan/declarations.hlsl"
```

Comment:
declarations (we use structs/functors now) and typeless arithmetic (we have templated binops now) probably not needed

Reply:
b-gon